AmplifyPlugins/Auth/Tests/AWSCognitoAuthPluginUnitTests/HubEventTests/AuthHubEventHandlerTests.swift

```diff
@@ -136,12 +136,12 @@ class AuthHubEventHandlerTests: XCTestCase {
             }
         }
 
-        _ = plugin.deleteUser { result in
-            if case .failure(let error) = result {
-                XCTFail("Received failure with error \(error)")
-            }
+        do {
+            _ = try await plugin.deleteUser()
```

Comment:
Is `_ =` needed?  `deleteUser()` no longer returns a value, correct?

Reply:
Good catch, code updated.